code/messaging/build.gradle

```diff
@@ -182,11 +207,12 @@ dependencies {
     // where Objenesis 2 requires minimum Android API 26
     androidTestImplementation 'org.powermock:powermock-module-junit4:1+'
     androidTestImplementation 'com.fasterxml.jackson.core:jackson-databind:2.9.9'
-    androidTestImplementation 'com.adobe.marketing.mobile:edgeidentity:1.0.0'
+    androidTestImplementation('com.adobe.marketing.mobile:edgeidentity:1.0.0')
+    androidTestImplementation('com.adobe.marketing.mobile:edge:1.1.1')
 }
 
 afterEvaluate {
-    generatePhoneReleaseBuildConfig.enabled = false
+    generatePhoneReleaseBuildConfig.enabled = true
```

Comment:
I was wondering why do we need to enable generating BuildConfig is there any other way we can find if the code is executing as part of unit test?
If we have no other option than BuildConfig then remove the `afterEvaluate` block, since the default value is true.

Reply:
the BuildConfig is used to setup the correct OffersConfig for e2e functional tests. I was unable to use mocking or setters to modify the OffersConfig as its created as private var within InAppNotificationHandler. I'll remove the afterEvaluate block and will see if there is a better way to resolve this for a future release.